helpers/misk.sh

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2019 Denis Gavrilyuk <karpa4o4@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Downloads files from the Web non-interactively.
+# Globals:
+#     PIEMAN_UTILS_DIR
+#     PYTHON
+# Arguments:
+#     Wget argument string.
```

Comment:
`Wget argument string` says nothing. Replace it for `URL`.

Reply:
But in addition to the _URL_ argument, _quite_ and _output-file_ are also passed to the function.

Reply:
Right, so let's do something like the following
```
wget options (only -O and -q are supported)
URL
```